mlir/include/mlir/Dialect/Bufferization/Transforms/BufferUtils.h

```diff
@@ -73,33 +73,35 @@ class BufferPlacementAllocs {
   AllocEntryList allocs;
 };
 
+/// Finds a common dominator for the given value while taking the positions
+/// of the values in the value set into account. It supports dominator and
+/// post-dominator analyses via template arguments.
```

Comment:
The "If no common dominator can be found, this function will return nullptr" part (generalized) is a bit lost after this merge. Perhaps add it back here?